athena-aws-cmdb/athena-aws-cmdb.yaml

```diff
@@ -10,7 +10,7 @@ Metadata:
     Labels:
       - athena-federation
     HomePageUrl: 'https://github.com/awslabs/aws-athena-query-federation'
-    SemanticVersion: 2022.4.1
+    SemanticVersion: 2022.7.0
```

Comment:
2022.7.1 (iteration will start at 1)

Reply:
also need to update the version in pom files. take a look at https://github.com/awslabs/aws-athena-query-federation/pull/665/files (usually just increment the last number)

Reply:
will do

Reply:
fixed